causal-inference-for-the-brave-and-true/04-Graphical-Causal-Models.ipynb

```diff
@@ -1143,11 +422,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In the following graph, conditioning on X1 and X2, or, SAT and family income, is sufficient to close all backdoor paths between the treatment and the outcome. In other words, $(Y_0, Y_1) \\perp T | X1, X2$. So even if we can't measure all common causes, we can still attain conditional independence if we control for measurable variables that mediate the effect of the unmeasured on the treatment. One quick note, we also have $(Y_0, Y_1) \\perp T | X1, U$, but since we can't observe U, we can't condition on it. \n",
+    "In the following graph, conditioning on X1 and X2, or SAT and family income, is sufficient to close all backdoor paths between the treatment and the outcome. In other words, $(Y_0, Y_1) \\perp T | X1, X2$. So even if we can't measure all common causes, we can still attain conditional independence if we control for measurable variables that mediate the effect of the unmeasured on the treatment. One quick note, we also have $(Y_0, Y_1) \\perp T | X1, U$, but since we can't observe U, we can't condition it. \n",
```

Comment:
Again, `condition` need the preposition `on` here. 
```suggestion
    "In the following graph, conditioning on X1 and X2, or SAT and family income, is sufficient to close all backdoor paths between the treatment and the outcome. In other words, $(Y_0, Y_1) \\perp T | X1, X2$. So even if we can't measure all common causes, we can still attain conditional independence if we control for measurable variables that mediate the effect of the unmeasured on the treatment. One quick note, we also have $(Y_0, Y_1) \\perp T | X1, U$, but since we can't observe U, we can't condition on it. \n",
```